index.html

```diff
@@ -3,10 +3,10 @@
 <head>
     <meta charset="utf-8">
     <title>JavaScript Loops</title>
-    <link href="/Users/ben/dev/learningth/style.css" rel="stylesheet">
+    <link href="/home/venomousmonkey/dev/treehouse/style.css" rel="stylesheet">
```

Comment:
Try a relative path.
You dont want to include info that is specific to your file system.
Like if I download the project, this would fail for me.

Reply:
How can I use a relative path?

Reply:
Changed to a relative path.